Reasonable.md

```diff
@@ -0,0 +1,25 @@
+# Reasonable
+
+To be reasonable is to be ALL (conjuctive) of:
+* rational - to hold logically consistent beliefs and values
+* agnostic - to have no faith
+* corrigible - able to be corrected
+* fallible - capable of making mistakes
+* cooperative - willing to work together for mutual benefit
```

Comment:
Nitpicking as per [dmc's comment](https://github.com/Vulcan-Academy/Vulcan-Academy.github.io/pull/35#issuecomment-366315420) - is it strictly always rational to cooperate?

Reply:
It is often reasonable to not cooperate but that is a different question than whether it is reasonable to be cooperative

Reply:
Yes, more nitpicking please.